### PR TITLE
Use Subclass Checkbox

### PR DESCRIPTION
**Pull request by guergana**
Not sure if more tests are needed. Seems they are all included in the previous commit.

### CODE REVIEW

File: src/components/SubclassCheckbox.vue [hunk 1]
```diff
@@ -5,7 +5,7 @@
 			type="checkbox"
 			id="include-subclasses"
 			:checked="isChecked"
-			@change="$emit( 'subclass-check', $event )"
+			@change="$emit( 'subclass-check', $event.target.checked )"
```
Comment (guergana) [inline src/components/SubclassCheckbox.vue:8]: We don't need the whole event. We just just emit what we need.

File: src/components/QueryCondition.vue [hunk 2]
```diff
@@ -32,6 +32,7 @@
 					:error="valueError"
 					:datatype="datatype"
 				/>
+				<SubclassCheckbox @subclass-check="setSubclasses" :isChecked="subclasses(conditionIndex)" />
```
Comment (micgro42) [inline src/components/QueryCondition.vue]: This should only be shown for properties with datatype `wikibase-item`

File: src/components/SubclassCheckbox.vue [hunk 3]
```diff
@@ -5,7 +5,7 @@
 			type="checkbox"
 			id="include-subclasses"
```
Comment (micgro42) [inline src/components/SubclassCheckbox.vue]: Now all these checkboxes have the same id, which means if I have two conditions and I click the label of the checkbox in the second condition, then the checkbox in the first condition is toggled.

**Review by micgro42 — CHANGES_REQUESTED** [review]

File: src/components/SubclassCheckbox.vue (hunk 3 above)
Comment (micgro42) [reply src/components/SubclassCheckbox.vue]: Something like `:id="id"`
and 
```js
data() {
    return {
        id: `include-subclasses-${Math.random()}`,
    };
},
```
should be good enough

File: src/components/QueryCondition.vue (hunk 2 above)
Comment (guergana) [reply src/components/QueryCondition.vue]: Oh, I understood it was only checked when datatype is `wikibase-item`  but always shown? Will update.

Comment (guergana) [reply src/components/QueryCondition.vue]: addressed!

File: src/components/SubclassCheckbox.vue (hunk 3 above)
Comment (guergana) [reply src/components/SubclassCheckbox.vue]: addressed!

File: src/components/SubclassCheckbox.vue [hunk 4]
```diff
@@ -3,9 +3,9 @@
 		<input
 			class="querybuilder__include-subclasses-checkbox"
 			type="checkbox"
-			id="include-subclasses"
+			:id="id"
 			:checked="isChecked"
-			@change="$emit( 'subclass-check', $event )"
+			@change="$emit( 'subclass-check', $event.target.checked )"
 		/>
 		<label for="include-subclasses">
```
Comment (micgro42) [inline src/components/SubclassCheckbox.vue]: You have to use the dynamic id here as well :)

**Review by micgro42 — CHANGES_REQUESTED** [review]
Almost there!

**Review by micgro42 — APPROVED** [review]
\o/

**Pull request merged**